docs/rules/jsx-no-bind.md

```diff
@@ -151,6 +151,35 @@ class Foo extends React.Component {
 
 A more sophisticated approach would be to use something like an [autobind ES7 decorator](https://www.npmjs.com/package/core-decorators#autobind) or [property initializers](https://facebook.github.io/react/blog/2015/01/27/react-v0.13.0-beta-1.html#autobinding).
 
+### React Hooks
+
+Functional components are often used alongside hooks, and the most trivial case would occur if your callback is completely independent from your state. In this case, the solution is as simple as moving the callback out of your component:
+
+```jsx
+const onClick = () => {
+  console.log("Independent callback");
+};
+const Button = () => {
+  return (
+    <button onClick={onClick}>Label</button>
+  );
+};
+```
+
+Otherwise, the idiomatic way to avoid redefining callbacks on every render would be to memoize them using the [`useCallback`](https://reactjs.org/docs/hooks-reference.html#usecallback) hook:
+
+```jsx
+const Button = () => {
+  const [text, setText] = useState("Before click");
+  const onClick = useCallback(() => {
+    setText("After click");
+  }, [text]); // Array of dependencies for which the memoization should update
```

Comment:
since `setText` is what's called, that's the dependency here
```suggestion
  }, [setText]); // Array of dependencies for which the memoization should update
```

Reply:
Hah, good catch. I knew I should have run this. :)